qemu/tests/block_copy.py

```diff
@@ -298,7 +297,10 @@ def action_when_start(self):
                 bg = utils.InterruptedThread(fun)
                 bg.start()
                 if bg.isAlive():
-                    self.processes.append(bg)
+                    try:
```

Comment:
and join mean wait for the process finished that mean sub process not works in parallel not step by step. so I don't know why update it, case changed?